README.md

```diff
@@ -1,18 +1,22 @@
 # Selfauth
-self-hosted auth_endpoint using simple login mechanism
 
-# Warning: while Selfauth will work with old versions of PHP, the ability to get sufficiently random strings was not added until version 5.6.  While older versions are not completely insecure, it is strongly recommended you upgrade to a newer version of PHP.
+Selfauth is a self-hosted [Authorization Endpoint](https://indieweb.org/authorization-endpoint) used to login with a personal URL (as [Web sign-in](http://indieweb.org/Web_sign-in)) via [IndieAuth](https://indieweb.org/IndieAuth). See [How it works](#how-it-works) for more.
 
-# Warning: currently selfauth only supports authentication, not authorization.  Meaning scopes will not work yet.  If you need access to these features, it is advisable to use something else for now.
 
-Setup
------
+## Warnings
+
+- While Selfauth will work with old versions of PHP, the ability to get sufficiently random strings was not added until version 5.6. While older versions are not completely insecure, **it is strongly recommended you upgrade to a newer version of PHP**.
```

Comment:
It isn’t just random strings anymore, also things like `hash_equals`. And possibly the `password_`-functions if we end up using those.

Can we have a more general description that for PHP versions below 5.6 we can’t use PHP’s security features and that we cannot guarantee the same full security even though Selfauth will still be functional?